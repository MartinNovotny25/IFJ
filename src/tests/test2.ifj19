def main():
    a = 1.9
    a = a * a
    return a
