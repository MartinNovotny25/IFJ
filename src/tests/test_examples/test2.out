TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(DOUBLE, 1.900000)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(IDENTIFIKATOR)
TOKEN(*)
TOKEN(IDENTIFIKATOR)
TOKEN(EOL)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(EOF)
